extensions/vertx-http/deployment/src/main/java/io/quarkus/devui/deployment/menu/ContinuousTestingProcessor.java

```diff
@@ -93,7 +96,14 @@ private void registerStartMethod(LaunchModeBuildItem launchModeBuildItem) {
                 if (testSupport.isStarted()) {
                     return false; // Already running
                 } else {
-                    testSupport.start();
+                    CompletableFuture<Void> start = CompletableFuture.runAsync(() -> testSupport.start());
```

Comment:
don't use runAsync, it will run on a F/J thread, and it can have terrible consequences if your parallelism is low.